azure_arc_servers_jumpstart/docs/gcp_terraform_windows.md

```diff
@@ -6,23 +6,39 @@ The following README will guide you on how to use the provided [Terraform](https
 
 * Clone this repo
 
-* [Generate SSH Key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) (or use existing ssh key) 
-
-* Google Cloud account - [create a free trial account](https://cloud.google.com/free)
+* [Generate SSH Key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) (or use existing ssh key)
 
 * [Install Terraform >=0.12](https://learn.hashicorp.com/terraform/getting-started/install.html)
 
+* Google Cloud account with billing enabled - [Create a free trial account](https://cloud.google.com/free). To create Windows Server virtual machines, you must upgraded your account to enable billing. Click Billing from the menu and then select Upgrade in the lower right.
+
+    ![](../img/gcp_windows/29.png)
+
+    ![](../img/gcp_windows/30.png)
+
+    ![](../img/gcp_windows/32.png)
+
+    ***Disclaimer*** - **Be sure to completely delete any resources created as part of the tutorial in order to prevent unexpected charges. The tutorial provides instructions on how to delete everything but double check so you don't get charged inadvertently.**
```

Comment:
Suggestion for a cleaner wording:

***Disclaimer*** - **To prevent unexpected charges, please follow the  "Delete the deployment" section at the end of this README**